frameworks/digital-outcomes-and-specialists-5/questions/declaration/contactEmail.yml

```diff
@@ -0,0 +1,3 @@
+name: Contact email
+question: What is the contact’s email address?
+type: text
```

Comment:
The DOS5 validator will need to be updated to make sure this field is validated to be an email address.

Reply:
:+1: added to the story checklist